java-squid/src/main/java/org/sonar/java/resolve/JavaType.java

```diff
@@ -228,6 +230,9 @@ public boolean isSubtypeOf(String fullyQualifiedName) {
 
     @Override
     public boolean isSubtypeOf(Type superType) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "isSubtypeOf" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)